guard.cpp

```diff
@@ -141,6 +146,8 @@ int main(int argc, char** argv)
                      "Clears GUARD states for all resources");
         app.add_flag("-p, --listresolvedrecords", listResolvedGuardRecords,
                      "Listing of resolved GUARDed resources");
+        app.add_flag("-e, --eraseall", eraseall,
+                     "Erase all the GUARDed resources");
```

Comment:
- Add proper difference between `-e` vs `-r` in the help section and README. 
  - I think in English `clear` and `erase` will mean the same.
- Do you need to communicate about the new changes of the clear option with all stakeholders (whoever using the guard tool in their component? For example MFG, CI, TestTeam, etc) on this?
- (Just checking) Do we need an option to just delete (remove from GUARD file) alone all resolved records?

Reply:
Will add the details in README. Yes, we need to communicate to test team. Once this is merged will do that.
I don't think there is any requirement to just remove invalid guard records.